github CI always fail

github action CI  always failed since 8.7 as same reason
CI error log:
Error: -08 11:00:45.044 [GeneralDMLIT-ShardingSphere-Proxy] : [ERROR] 2021-08-08 11:00:44.859 [main-SendThread()] o.a.z.client.StaticHostProvider - Unable to resolve address: zk:2181
IT error log:
no suitable method found for createDataSource(java.util.Map<java.lang.String,javax.sql.DataSource>,java.util.Set<org.apache.shardingsphere.shadow.api.config.ShadowRuleConfiguration>,<nulltype>,java.lang.String)

--- a/examples/shardingsphere-jdbc-example/other-feature-example/shadow-example/shadow-raw-jdbc-example/src/main/java/org/apache/shardingsphere/example/shadow/table/raw/jdbc/config/ShadowDatabasesConfiguration.java
+++ b/examples/shardingsphere-jdbc-example/other-feature-example/shadow-example/shadow-raw-jdbc-example/src/main/java/org/apache/shardingsphere/example/shadow/table/raw/jdbc/config/ShadowDatabasesConfiguration.java
@@ -17,11 +17,10 @@
 
 package org.apache.shardingsphere.example.shadow.table.raw.jdbc.config;
 
+import org.apache.shardingsphere.driver.api.ShardingSphereDataSourceFactory;
 import org.apache.shardingsphere.example.config.ExampleConfiguration;
 import org.apache.shardingsphere.example.core.api.DataSourceUtil;
-import org.apache.shardingsphere.infra.database.DefaultSchema;
 import org.apache.shardingsphere.shadow.api.config.ShadowRuleConfiguration;
-import org.apache.shardingsphere.driver.api.ShardingSphereDataSourceFactory;
 
 import javax.sql.DataSource;
 import java.sql.SQLException;
@@ -37,6 +36,6 @@ public final class ShadowDatabasesConfiguration implements ExampleConfiguration 
         Map<String, DataSource> dataSourceMap = new HashMap<>(2, 1);
         dataSourceMap.put("ds", DataSourceUtil.createDataSource("demo_ds"));
         dataSourceMap.put("ds_0", DataSourceUtil.createDataSource("shadow_demo_ds"));
-        return ShardingSphereDataSourceFactory.createDataSource(dataSourceMap, Collections.singleton(shadowRuleConfig), null, DefaultSchema.LOGIC_NAME);
+        return ShardingSphereDataSourceFactory.createDataSource(dataSourceMap, Collections.singleton(shadowRuleConfig), null);
     }
 }

